dag_status.py

```diff
@@ -0,0 +1,38 @@
+import time
+import requests
+
+def get_latest_state():
+    url = "http://localhost:8080/api/v1/dags/lakeFS_workflow/dagRuns"
+    username="admin"
+    password="admin"
```

Comment:
These could be params to the script, and then set these as their defaults.

(Not required)

Reply:
These would be  the default while astro setsup all the time so kept default.